### ISSUE TITLE
experiment with removing popup.css from steal, the initial load has no styles

### ISSUE THREAD

**Issue by shane-tomlinson** [mozilla/persona#365]
This makes the content jump around.

**Comment by lloyd** [comment]
I would lobby for this being higher priority (first thing I noticed, mike hanson mentioned it too.  it's a nit, but it's very in your face.)  adding needs-triage label so we can duke it out

**Comment by michaelrhanson** [comment]
as would I, it looks ugly for about 250ms on a coffeeshop network

**Comment by shane-tomlinson** [comment]
@lloyd - no need to duke it out, it was a relatively straight forward fix.  I'd like to get rid of all of StealJS and even JMVC - we aren't really using it what it is meant for and so far it is just adding a lot of extra cruft.

Merge 24d9b3641b596131abda9e5a601fc5dda8d6f343 brings this into dev.

Checkins 8d99e0311b2bb0f8e569f70c3a6de8a3faaacac7 and 34d16808fbbf3e5b48eef0aaeca92c37e81960bc are the individual checkins.  Be careful just cherry picking those checkins though, I rebased the branch I was working in from dev just before I did the merge, so I am not sure how that affects the contents of each previous checkin.
